docs/ispc.rst

```diff
@@ -701,6 +701,10 @@ recovery. Such users must update their code to handle ``SIGILL`` instead. The
 change improves predictability and eliminates reliance of the dispatcher on the
 C standard library.
 
+The ``--darwin-version-min`` option was added to specify the minimum deployment
+target version for macOS and iOS applications which is required by new linker
```

Comment:
It's not required by new linker, but the new linker will issue a warning for that.

Reply:
Fixed